### PR TITLE
feat: command controller for mediator reconnect

### LINKED ISSUE #2000: mediator command to reconnect router
to re-establish lost router connection

### PR DESCRIPTION
**Pull request by sudeshrshetty**
- added JS, command, REST controller for mediator reconnect by sending
noop message to router
- Closes #2000

Signed-off-by: sudesh.shetty <sudesh.shetty@securekey.com>

### CODE REVIEW

File: pkg/framework/aries/default.go [hunk 1]
```diff
@@ -10,6 +10,8 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/messagepickup"
+
```
Comment (rolsonquadras) [inline pkg/framework/aries/default.go:14]: extra line

File: pkg/controller/command/mediator/command.go [hunk 2]
```diff
@@ -12,6 +12,8 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/hyperledger/aries-framework-go/pkg/client/messagepickup"
+
```
Comment (rolsonquadras) [inline pkg/controller/command/mediator/command.go:16]: extra line

File: pkg/framework/aries/default.go (hunk 1 above)
Comment (sudeshrshetty) [reply pkg/framework/aries/default.go]: fixed

File: pkg/controller/command/mediator/command.go (hunk 2 above)
Comment (sudeshrshetty) [reply pkg/controller/command/mediator/command.go]: fixed

**Pull request merged**